core/templates/mari.yml

```diff
@@ -89,3 +89,15 @@ strings:
     tk-mari_project_asset_project_name:     '{Step}_{AssetType}-{Asset}-{mari.project_name}'
     tk-mari_sequence_asset_project_name:    '{Sequence}_{Step}_{AssetType}-{Asset}-{mari.project_name}'
     tk-mari_shot_asset_project_name:        '{Sequence}_{Shot}_{Step}_{AssetType}-{Asset}-{mari.project_name}'
+
+    # SHOT ASSET TEXTURE NAME
+    tk-mari_project_texture_name:           '{Step}_{mari.project_name}@channel_name'
```

Comment:
You can use additional referencing, like:
`tk-mari_project_texture_name:           '@tk-mari_project_project_name@channel_name'`
if you want to avoid repetition.